### ISSUE TITLE
Copy Mod List as text 

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge#6577]
Talking on accessibility heop this is a nice way to help colleagues debug patches

**Comment by baconpaul** [comment]
I should have made it clearer sorry!
But here's the details, all in ModulationEditor.cpp
There's a class ModulationListContents which does al the filtering. It maintains at any moment std::vector<Datum> dataRows and the Datum contains everything you need to build the list; It also as a DataRowEditor which turns it into strings and paints it

So the issue really is something like

1. Add std::string datumToString(const Datum &d) and code it
2. Add a 'copy' button to the side
3. Do a 
std::ostringstream oss
foreach( d : dataRows ) 
   oss << datumToString(d) << "\n";
juce::ClipboardGoobits::copy(oss.str());
